tests/helpers/reference_metrics.py

```diff
@@ -202,6 +202,28 @@ def _calibration_error(
     return loss
 
 
+def _sk_ergas(preds, target, ratio, reduction):
+    # reshape to (batch_size, channel, height*width)
+    B, C, H, W = preds.shape
+    sk_preds = preds.reshape(B, C, H * W)
+    sk_target = target.reshape(B, C, H * W)
+    # compute rmse per band
+    diff = sk_preds - sk_target
+    sum_squared_error = torch.sum(diff * diff, dim=2)
+    rmse_per_band = torch.sqrt(sum_squared_error / (H * W))
+    mean_target = torch.mean(sk_target, dim=2)
+    # compute ergas score
+    ergas_score = 100 * ratio * torch.sqrt(torch.sum((rmse_per_band / mean_target) ** 2, dim=1) / C)
```

Comment:
again lets follow pep8 standards
```suggestion
    b, c, h, w = preds.shape
    sk_preds = preds.reshape(b, c, h * w)
    sk_target = target.reshape(b, c, h * w)
    # compute rmse per band
    diff = sk_preds - sk_target
    sum_squared_error = torch.sum(diff * diff, dim=2)
    rmse_per_band = torch.sqrt(sum_squared_error / (h * w))
    mean_target = torch.mean(sk_target, dim=2)
    # compute ergas score
    ergas_score = 100 * ratio * torch.sqrt(torch.sum((rmse_per_band / mean_target) ** 2, dim=1) / c)
```